frontend/public/components/catalog/catalog-item-details.jsx

```diff
@@ -77,7 +77,7 @@ export class CatalogTileDetails extends React.Component {
               </PropertiesSidePanel>
               <div className="co-catalog-page__overlay-description">
                 <SectionHeading text="Description" />
-                {tileDescription && <SyncMarkdownView content={tileDescription} />}
+                {modalDescription && <SyncMarkdownView content={modalDescription} />}
```

Comment:
Let's bring back the original implementation and override if there is a `modalDescription`, otherwise the other tiles don't display a description at all.
```suggestion
                {!modalDescription && tileDescription && <p>{tileDescription}</p>}
                {modalDescription && <SyncMarkdownView content={modalDescription} />}
```

Reply:
All set, thank you!